api.py

```diff
@@ -54,32 +54,16 @@ def get(self, token, url):
 
         return (response.status_code, response.json()['response'])
 
-    # can't use this because POST requests not accepted
-    def upload_pdf_wrong(self, token, url, filename):
-        """ Make authenticated POST request for uploading PDF
-        and return response status """
+    def put(self, token, url, filename):
+        """ Make authenticated PUT request for uploading file
+        and return response status and data"""
 
-        files = {
-            "file": (f'{filename}',
-                    open(f'{filename}', 'rb'),
-                    "application/pdf",
-                    {'Authorization': f'Bearer {token}'})
-        }
-        response = requests.post(url, files=files)
-        print(response.raise_for_status())
-
-        return (response.status_code)
-
-    # TODO: Figure out what is still wrong
-    # "response":{"content_file":["No file was submitted"]} -> Why?
-    def upload_pdf(self, token, url, filename):
-        """ Make authenticated POST request for uploading PDF
-        and return response status """
-        
         headers = {
-            'Content-Type': 'multipart/form-data',
             'Authorization': f'Bearer {token}',
         }
-        response = requests.put(url, data=open(f'{filename}', 'rb'), headers=headers)
+        files = {
```

Comment:
This is the wrong place to get the file.

I think this would be better if you pass the file into the function. What happens if you want to use put but without a file?